### DISCUSSION TITLE
Not able to show the updated value of  variable in Ckeditor due to wire:ignore on the parent Div

### DISCUSSION THREAD

**Discussion by MAdnanNaeem** [livewire/livewire#5402] [1. Help]
Hi there ! I'm using Ckeditor5 classic in my laravel livewire project and I'm using Ckeditior on bootstrap model. I used "wire:ignore"  on parent div of Text-area. and When i try to edit the note_data. It do not show up on the Ckeditior due to Wire:ignore. Can anyone please help me out.
```
<div class="col-md-6 col-lg-12 col-xl-12">
                            <div wire:ignore class="form-group">
                                @error('patient_notes')
                                    <span class="error text-danger">{{ $message }}</span>
                                @enderror
                                <textarea class="form-control" data-note="@this" wire:model="patient_notes" id="note"> <?= $patient_notes ?></textarea>
</div></div>
```

**Comment by joshhanley** [comment]
@MAdnanNaeem if you make use of `wire:ignore` then anything inside of it will be ignored by Livewire (including it's own directives).

When using a third party javascript library, you generally need to use `wire:ignore` if the library modifies the DOM, so Livewire doesn't wipe it out. In those instances, you need to use Livewire's javascript API or AlpineJS to communicate the changes between your library and Livewire. See here for details https://laravel-livewire.com/docs/2.x/alpine-js#sharing-state

Hope this helps!
